department-of-veterans-affairs/va.gov-cms
# VACMS-8861: display error message if office hours comment contains ti…

…me zone

## Description

Closes #8861 

## Testing done

Visual Testing

## Screenshots

Strings containing time zones as substrings are allowed (Example: the word established contains est)
<img width="1185" alt="Screen Shot 2022-05-20 at 2 34 13 PM" src="https://user-images.githubusercontent.com/21045418/169591811-7c02e28d-45bd-4fa3-8d9d-a09c426efbb6.png">

Actual time zones are not allowed
<img width="794" alt="Screen Shot 2022-05-20 at 2 41 26 PM" src="https://user-images.githubusercontent.com/21045418/169592250-b97a72c8-e170-49a4-b657-5e9eac91bd94.png">

## QA steps

As an admin user or content editor
1. Edit a VAMC Facility Non-clinical Service (in this example a register for care node) /node/46300/edit
2. Expand the service location hours field and try adding some time zones into the comments (EST, ET, CST, CT, MST, MT, PST, PT) case does not matter
3. An error should be displayed and the office hours field(s) should be highlighted in red.

### Select Team for PR review


## Deployment

Once this code has been merged and deployed we will want to run the following script on production: `drush scr ./scripts/content/VACMS-8935-scrub-tz-in-hours-field-comment.php`
